models/POP/shell_scripts/cesm2_1/CESM2_1_setup_pmo

```diff
@@ -0,0 +1,763 @@
+./xmlchange STOP_N=$first_STOP_N
+./xmlchange CONTINUE_RUN=FALSE
+./xmlchange RESUBMIT=$resubmit
+
+./xmlchange PIO_TYPENAME=pnetcdf
+
+./xmlchange GLC_NCPL=1
+./xmlchange OCN_NCPL=1
+
+# These are archiving options that may be used.
+# You can turn the short/long term archivers on or off,
+# but these settings should be made in either event.
+
+./xmlchange DOUT_S_ROOT=${ARCHDIR}
+./xmlchange DOUT_S_SAVE_INTERIM_RESTART_FILES=TRUE
+
+if ($short_term_archiver == 'off') then
+   ./xmlchange DOUT_S=FALSE
+else
+   ./xmlchange DOUT_S=TRUE
+endif
+
+# level of debug output, 0=minimum, 1=normal, 2=more, 3=too much, valid values: 0,1,2,3 (integer)
+
+./xmlchange DEBUG=FALSE
+./xmlchange INFO_DBUG=0
+
+# ==============================================================================
+# Updating source files.
+#
+# Ideally, using DART would not require any modifications to the model source.
+# Until then, this script accesses sourcemods from a hardwired location.
+# If you have additional sourcemods, they will need to be merged into any DART
+# mods and put in the SourceMods subdirectory found in the 'CASEROOT' directory.
+# ==============================================================================
+
+echo ''
+echo 'Updating source files.'
+echo ''
+
+if (    -d     ~/${cesmtag}/SourceMods ) then
+   ${COPY} -r  ~/${cesmtag}/SourceMods/* ${CASEROOT}/SourceMods/
+else
+   echo "ERROR - No SourceMods for this case."
+   echo "DART requires modifications to several src files."
+   echo "Download the appropriate files for CESM2_1 from:"
+   echo "http://www.image.ucar.edu/pub/DART/CESM"
+   echo "untar these into your HOME directory - they will create a"
+   echo "~/${cesmtag}  directory with the appropriate SourceMods structure."
+   exit -4
+endif
+
+# ==============================================================================
+# Updating the run script to reflect queue and wallclock
+# ==============================================================================
+
+echo ''
+echo 'Updating the run script to set wallclock and queue.'
+echo ''
+
+./xmlchange --subgroup case.st_archive --id JOB_WALLCLOCK_TIME --val $timewall
+./xmlchange --subgroup case.run --id USER_REQUESTED_WALLTIME --val $timewall
+./xmlchange --subgroup case.run --id JOB_WALLCLOCK_TIME --val $timewall
+
+# ==============================================================================
+# Setting up the case.
+#
+# This creates the EXEROOT and RUNDIR directories.
+# ==============================================================================
+
+echo ''
+echo 'Setting up the case ...'
+echo 'This creates the EXEROOT and RUNDIR directories.'
+echo ''
+
+./case.setup
+
+if ( $status != 0 ) then
+   echo "ERROR: Case could not be set up."
+   exit -2
+endif
+
+# ==============================================================================
+# Modifying namelist templates for each instance.
+# ==============================================================================
```

Comment:
Since there is only 1 instance, do you want to mention that the reason there is a loop is to minimize the differences between this script and the hybrid setup script. So if you get familiar with this one, the assimilation setup will be pretty familiar.